src/server/config/env/runTime.js

```diff
@@ -118,7 +118,7 @@ const runTime = [
   // locale folder env level
   {
     name: 'ONE_CLIENT_LOCALE_FILENAME',
-    // E1, E2, E3 (E3 use the moduleName)
+    // dev, qa, staging and production (in production use the moduleName)
```

Comment:
per the line below, `"qa"` and `"integration"` or `undefined` are the valid values. I think this comment may just be removed altogether, unless we make some functionality changes

Reply:
thank you, i've updated this